### PR TITLE
[Merged by Bors] - cor/nix-readme

### PR DESCRIPTION
**Pull request by cor**
## Description
Here we add general nix documentation, so that people know how to build and run our packages.

## Checklist

- [x] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,4 +29,58 @@
...
+Once you have `nix` or `NixOS` installed, you should enable the following features:
+```nix
+{
+  nix = {
+    useSandbox = "relaxed";
+    extraOptions = ''
+      experimental-features = nix-command flakes
+    '';
+  };
+}
```
Comment (hussein-aitlahcen) [inline README.md:49]: For non-nixos, probably easier to set `~/.config/nix/nix.conf` with 
```
experimental-features = nix-command flakes
sandbox = relaxed
```

**Review by hussein-aitlahcen — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cor) [reply README.md:49]: Will add 👍🏻

Comment (cor) [reply README.md:54]: added

**Review by 0xBrainjar2 — APPROVED** [review]

**Review by hussein-aitlahcen — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Pull request closed without merging**